python/bacalhau_sdk/job_store.py

```diff
@@ -0,0 +1,152 @@
+from bacalhau_sdk.orchestrator_service import OrchestratorService
+from bacalhau_apiclient.models.api_put_job_response import (
+    ApiPutJobResponse as PutJobResponse,
+)
+from bacalhau_apiclient.models.api_put_job_request import (
+    ApiPutJobRequest as PutJobRequest,
+)
+from bacalhau_apiclient.models.api_stop_job_response import (
+    ApiStopJobResponse as StopJobResponse,
+)
+from bacalhau_apiclient.models.api_list_job_executions_response import (
+    ApiListJobExecutionsResponse as ListJobExecutionsResponse,
+)
+from bacalhau_apiclient.models.api_list_job_results_response import (
+    ApiListJobResultsResponse as ListJobResultsResponse,
+)
+
+from bacalhau_apiclient.models.api_list_job_history_response import (
+    ApiListJobHistoryResponse as ListJobHistoryResponse,
+)
+from bacalhau_apiclient.models.api_list_jobs_response import (
+    ApiListJobsResponse as ListJobsResponse,
+)
+from bacalhau_apiclient.models.api_get_job_response import (
+    ApiGetJobResponse as GetJobResponse,
+)
+
+from bacalhau_sdk.config import init_config
+
+
+class JobStore:
+    def __init__(self):
+        self.orchestrator_service = OrchestratorService(config=init_config())
+
+    def put(self, request: PutJobRequest) -> PutJobResponse:
+        """Puts Job On Bacalhau Network
+
+        Args:
+            request (PutJobRequest): A request to put a job to bacalhau network. It encapsulates the job model.
+
+        Returns:
+            PutJobResponse: Once job is successful put on bacalhau netowork, this returns the job details.
+        """
+        return self.orchestrator_service.put_job(request)
+
+    def stop(self, job_id: str, reason: str = None) -> StopJobResponse:
```

Comment:
I forgot about the standards in python, but should they be `stop` or `Stop`?

Reply:
Python is snake case so `stop`.